db/migrate/20210207162225_move_kv_ids_to_model.rb

```diff
@@ -0,0 +1,79 @@
+class MoveKvIdsToModel < ActiveRecord::Migration[6.0]
+  def up
+    create_table :kvs do |t|
+      t.string :name, null: false
+      t.integer :pbs_id, null: false
+      t.string :locale, null: false
+
+      t.index :pbs_id, unique: true
+    end
+    add_foreign_key :units, :kvs, validate: false, column: :kv_id, primary_key: :pbs_id
```

Comment:
```suggestion
    add_foreign_key :units, :kvs, validate: true, column: :kv_id, primary_key: :pbs_id
```
if we are sure, that the import was always correct, we could go for this